src/onCLS.ts

```diff
@@ -66,9 +65,8 @@ export const onCLS = (onReport: CLSReportCallback, opts?: ReportOpts) => {
       let report: ReturnType<typeof bindReporter>;
 
       let sessionValue = 0;
-      let sessionEntries: PerformanceEntry[] = [];
+      let sessionEntries: LayoutShift[] = [];
```

Comment:
minor type bug that's an explicit error now with these changes (on `sessionEntries.push(entry)` below), but no functional change